### ISSUE TITLE
Hide Balances

### ISSUE THREAD

**Issue by NashJames** [Nault/Nault#413]
**Is your feature request related to a problem? Please describe.**
Two purposes:
- Checking transactions without having someone view your total balances over your shoulder
- Bitvavo now requests screen recordings of you accessing your account, to verify you own the account. I would rather not share the exact balances of my accounts to their record.

**Describe the solution you'd like**
A simple toggle button, to replace any nano or fiat values with a set number of asterisks. Or something similar

**Additional context**
Ideally this would including the individual account pages and their transaction lists too. Not just the total balance in the corner

**Comment by Joohansson** [comment]
There are balances in many places in the wallet such as the account detail page, account history, the main menu balance widget, send screen, receive screen etc. Should they all be hidden from an app setting or just add a toggle button to the main widget? I guess all because there is nowhere to navigate without showing balances somewhere. But maybe the main widget toggle button could also trigger a global hide. I'm not sure what's the most user friendly here.

**Comment by NashJames** [comment]
I think a global hide would cover the majority of use cases, and would be a simpler implementation. The location of the toggle, main widget or settings page, is largely up to you. I suppose there is a chance of it getting frequent use, if someone wishes to only briefly view their balance for example, then hide it again.

**Comment by andrew-m-higgs** [comment]
Is anyone working on this?

I have implemented a solution using configure-app.
I did a transfer out while balances were hidden and saw that the balance of the account is visible there too. Will track that one down and fix it. 
I guess the amount being transferred should not be hidden. Easier to make sure you are sending the correct amount. 
Thoughts?

**Comment by geommr** [comment]
> I guess the amount being transferred should not be hidden. Easier to make sure you are sending the correct amount.

agreed. As far as I can tell, there are 4 places where the balance should/could be hidden: 
- the wallet card widget
- account details page
- list of accounts
- sending confirmation page

@andrew-m-higgs any progress on this? I could eventually work on it if not.

**Comment by andrew-m-higgs** [comment]
@geommr There is a pull request in for this (https://github.com/Nault/Nault/pull/476).

Regards
